_pages/clients/python.md

```diff
@@ -199,26 +198,36 @@ except Exception as error:
     print(error)
 {% endhighlight %}
 
-Once the algorithm has completed, `response.result` will contain the URI for the resulting file, which you can then download (or provide as input to another algorithm in a pipeline).
+Once the algorithm has completed, `response.result` will contain information about the dimensions of the bounding boxes for any detected faces, the size of the image, and the URI for the resulting file, which you can then download (or provide as input to another algorithm in a pipeline).
+
+Algorithms can create and store data in folders named with the algorithm name in the Algorithm Data collection. To access this folder from within an executing algorithm, the `.algo` shortcut can be used, as in the input example above. When accessing data from a client context, the algorithm author and name can be used along with the `.algo` shortcut to download data, in the format `data://.algo/author/algoName/folder/fileName`.
+{: .notice-info}
 
 ### Download the resulting file
 
-Verify that the file that you want to download exists, and try downloading it to a (new) local file:
+The URI included in the algorithm output uses the `.algo` shortcut, so we'll need to modify it slighly to download the file by adding the algorithm name and author:
+
+{% highlight python %}
+output_uri = response.result['images'][0]['output']
+download_uri = output_uri[:13] + `dlib/FaceDetection` + output_uri[13:]
+{% endhighlight %}
+
+Verify that the file that you want to download exists, and try downloading it to a new local file:
 
 {% highlight python %}
 # Download the file
-if client.file(response.result).exists() is True:
-    localfile = client.file(response.result).getFile()
+if client.file(download_uri).exists() is True:
+    local_file = client.file(download_uri).getFile()
 {% endhighlight %}
 
-This copies the file from your data collection and saves it as a file on your local machine, storing the filename in the variable `localfile`.
+This copies the file from your data collection and saves it as a file on your local machine, storing the filename in the variable `local_file`.
 
 Alternately, if you just need the binary content of the file to be stored in a variable, you can retrieve the remote file's content without saving the actual file:
 
 {% highlight python %}
 # Download contents of file as a string
-if client.file(response.result).exists() is True:
-    output = client.file(response.result).getBytes()
+if client.file(download_uri).exists() is True:
+    output = client.file(download_uri).getBytes()
 {% endhighlight %}
 
 This will get the image as binary data, saving it to the variable `input`, which might be useful when writing algorithms that are part of an image processing pipeline. For more image-manipulation tutorials, see the [Computer Vision Recipes]({{site.baseurl}}/tutorials/recipes/#computer-vision). 
```

Comment:
I think you meant to say "...the variable `output`..." instead of input

Reply:
I did mean input, because I was thinking about using this in the context of a pipeline or something where it was the input to the next algorithm. Maybe `image_data` would be better?